src/contexts/bridge.context.tsx

```diff
@@ -14,13 +15,19 @@ import { Bridge__factory } from "src/types/contracts/bridge";
 import { Erc20__factory } from "src/types/contracts/erc-20";
 import { BRIDGE_CALL_GAS_INCREASE_PERCENTAGE } from "src/constants";
 import { calculateFee } from "src/utils/fees";
+import tokenIconDefaultUrl from "src/assets/icons/tokens/erc20-icon.svg";
```

Comment:
We have to make sure that this URL is properly resolved once deployed.

Reply:
I have `npm run build && npm run start:prod` and confirmed that the file is properly served from `http://localhost/static/media/erc20-icon.dd72ab1b.svg`